src/ast/json.rs

```diff
@@ -1,13 +1,13 @@
-//! A hard-coded specification of JSON ASTs in a format editable by Sapling
+//! A hard-coded specification of Json ASTs in a format editable by Sapling
```

Comment:
I know this is pretty nitpicky of me, but I think `JSON` should be all-caps when it's not referring to the code...

Here's where the lines are:

### src/ast/test_json.rs
```
1://! Another representation of Json trees (where nodes own their children), useful for unit tests
16:    /// A Json array `[child1, child2, ..., childN]`.  Converts to [`Json::Array`]
18:    /// A Json object `{key1: child1, key2: child2, ..., keyN: childN}`.  Converts to a
21:    /// A Json 'String' literal
```

### src/ast/json.rs
```
1://! A hard-coded specification of Json ASTs in a format editable by Sapling
8:/// An enum to hold the different ways that a Json AST can be formatted
25:/// The sapling representation of the AST for a subset of Json (where all values are either 'true'
29:    /// The Json value for 'true'.  Corresponds to the string `true`.
31:    /// The Json value 'false'.  Corresponds to the string `false`.
33:    /// The Json value 'null'.  Corresponds to the string `null`.
35:    /// A Json array of multiple values.
36:    /// Corresponds to a string `[<v1>, <v2>, ...]` where `v1`, `v2`, ... are Json values.
38:    /// A Json object, represented as a map of [`String`]s to more Json values.
40:    /// ... are the keys, and `<v1>`, `<v2>`, ... are the corresponding Json values.  The `Ref`s
43:    /// A Json object field.  The first `Ref` must be a [`Str`](Json::Str), and the second is any
44:    /// Json object
46:    /// A Json string
51:    /// Return an iterator over all the possible chars that could represent Json nodes
```